tests/conftest.py

```diff
@@ -44,9 +46,10 @@ def entry_xml(self):
               </content>
             </entry>
         """
-        self.attributes["event_date_time"] = xsDateTime_format(
-            self.attributes["event_date_time"]
-        )
+        if isinstance(self.attributes['event_date_time'], datetime.datetime):
+            self.attributes["event_date_time"] = xsDateTime_format(
+                self.attributes["event_date_time"]
+            )
```

Comment:
Sorry! Didn't mean I don't want to be consistent. Just meant I forget to check when I wrap stuff!